tests/Predis/Command/Redis/ACL_Test.php

```diff
@@ -89,7 +89,7 @@ public function testSetUserCreatesACLUser(): void
     /**
      * @group connected
      * @return void
-     * @requiresRedisVersion >= 7.0.0
+     * @requiresRedisVersion == 7.0.0
```

Comment:
Let's use `<=` and `>=` if you want to isolate tests. With the current notation it won't run reliably.

Reply:
Then we need to add a support for this, since it's doesn't accept intervals

Reply:
Done